src/mlpack/methods/ann/layer/concatenate.hpp

```diff
@@ -0,0 +1,122 @@
+/**
+ * @file concatenate.hpp
+ * @author Atharva Khandait
+ *
+ * Definition of the Concatenate class that concatenate a constant matrix to
+ * the incoming data.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_METHODS_ANN_LAYER_CONCATENATE_HPP
+#define MLPACK_METHODS_ANN_LAYER_CONCATENATE_HPP
+
+#include <mlpack/prereqs.hpp>
+#include <mlpack/methods/ann/layer/layer_traits.hpp>
+
+namespace mlpack {
+namespace ann /** Artificial Neural Network. */ {
+
+/**
+ * Implementation of the Concatenate module class. The Concatenate module
+ * concatenates a constant given matrix to the incoming data.
+ * Note: Users need to use the Concat() function to provide the concat matrix.
+ *
+ * @tparam InputDataType Type of the input data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ * @tparam OutputDataType Type of the output data (arma::colvec, arma::mat,
+ *         arma::sp_mat or arma::cube).
+ */
+template <
+    typename InputDataType = arma::mat,
+    typename OutputDataType = arma::mat
+>
+class Concatenate
+{
+ public:
+  /**
+   * Create the Concatenate object using the specified number of output units.
+   *
+   * @param conat The matrix to be concatenated to the input.
```

Comment:
Looks like we are going with the ``Concat()``function, so we can remove the ``@param conat The matrix to be concatenated to the input.``.

Reply:
Oh, sorry I forgot to change this when you mentioned last time.